be/src/storage/rowset/segment_iterator.cpp

```diff
@@ -1096,6 +1301,12 @@ inline Status SegmentIterator::_read(Chunk* chunk, vector<rowid_t>* rowids, size
         chunk->check_or_die();
     }
 
+    if (_use_vector_index) {
+        for (uint32_t i = range.begin(); i < range.end(); i++) {
+            _first_rowids.push_back(i);
```

Comment:
Does `_first_rowids` need to be cleared after a invocation of `do_get_next` finished?

`_read` will be called multiple times for a invocation of `do_get_next`, and `do_get_next` will be called multiple times by `ChunkSource`.